crates/testlang/outputs/npm/package/build.rs

```diff
@@ -12,5 +12,5 @@ fn main() -> Result<()> {
     let output_dir =
         CargoWorkspace::locate_source_crate("testlang_npm_package")?.join("src/generated");
 
-    RuntimeGenerator::generate_product(&language, &input_dir, &output_dir)
+    RuntimeGenerator::generate_product(&language, &input_dir, &output_dir).map(|_| ())
```

Comment:
in `crates/solidity/testing/snapshots/bindings_output/contracts/super_scope/generated/0.4.16-failure.txt` it seems like we are trying (and failing) to bind keywords like `virtual` and `override` .. is this a bug? shouldn't they be excluded from binding?

Reply:
I see the same thing hapenning for `_;` in modifiers as well. Should this be also excluded?

Reply:
Both `virtual` and `override` were introduced in 0.6.0 so it makes sense the parser generated them as identifiers. They are being treated as modifier invocations, and of course the modifiers don't exist.

The `_;` is slightly similar, only that it's never parsed as a special keyword. In the `complete-builtins` branch I have [defined it](https://github.com/manastech/slang/blob/33ecc0552be48ef5040733dfc8f1df312c01faae/crates/solidity/outputs/cargo/slang_solidity/src/generated/bindings/generated/built_ins/0.8.26.sol#L82) as a global variable. I now realize this is not entirely correct, as `_` only has special meaning inside a modifier. I'll think of a better solution.

Reply:
Thanks for explaining.